deco/conc.py

```diff
@@ -113,8 +113,12 @@ def __init__(self, *args, **kwargs):
         if len(args) > 0 and type(args[0]) == types.FunctionType:
             self.setFunction(args[0])
         else:
-            self.__dict__.update({concurrent.params[i]: arg for i, arg in enumerate(args)})
-            self.__dict__.update({key: kwargs[key] for key in concurrent.params if key in kwargs})
+            self.__dict__.update(
+                dict([(concurrent.params[i], arg) for i, arg in enumerate(args)])
```

Comment:
instead of building another dict I think the better way of doing is passing a list of 2-tuples (key, value) .. ;)